app/models/event_stream.rb

```diff
@@ -66,6 +70,18 @@ def self.event_groups
     raise NotImplementedError, "event_groups must be implemented in a subclass"
   end
 
+  def ext_management_system_name
+    ext_management_system.try(:name)
+  end
+
+  def host_name
+    host.try(:name)
+  end
+
+  def vm_or_template_name
+    vm_or_template.try(:name)
+  end
+
```

Comment:
```suggestion
```

these are no longer necessary.

(and, these should all be done in sql so we won't have to join to these tables at all)